### PR TITLE
Add survey list with actions on homepage

### PR DESCRIPTION
**Pull request by AashnaNarang**
Closes https://github.com/AashnaNarang/SYSC4806/issues/67

- List of surveys on the homepage and actions
- If the survey is live, you can pick the first two options
- If the survey is not live, then you can only view metrics

image

Error handling if failed to update survey (this error shouldn't pop up once other tasks are done but added error handling just in case):
image

### CODE REVIEW

File: app/javascript/components/SurveyListActions.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            { isLive ? 
+              (
+                <Select
+                  labelId="demo-simple-select-label"
+                  id="demo-simple-select"
+                  value={""}
+                  label="Actions"
+                  onChange={handleChange}
+                >
+                  <MenuItem value={"respond"}>Respond to</MenuItem>
```
Comment (AndrewHall797) [inline app/javascript/components/SurveyListActions.js:82]: If its too much work than I would not bother, but if its not could we make these three distinct separate buttons? This is just a personal UI preference, but by no means crucial. Other than that it looks good to me, and again, if its too much of a pain  in the but I would not bother.

Comment (AashnaNarang) [reply app/javascript/components/SurveyListActions.js:82]: I think we should skip that and focus our time on fixing heroku!

Comment (AndrewHall797) [reply app/javascript/components/SurveyListActions.js:82]: kk sounds good

**Review by AndrewHall797 — APPROVED** [review]
LGTM

**Pull request merged**